userland/userland_generic.ld

```diff
@@ -20,14 +20,15 @@ MEMORY {
 }
 
 SECTIONS {
-    /* Text section, Code! */
-    .text :
+    /* Section for just the app crt0 header.
+     * This must be first so that the app can find it.
+     */
+    .crt0_header :
     {
-        _text = .;
+        _beg = .; /* Start of the app in flash. */
```

Comment:
Can we rename this to something understandable? They aren't charging by the character these days.